README.md

```diff
@@ -0,0 +1,160 @@
+# 📗 Table of Contents
```

Comment:
- [ ] OPTIONAL: I couldn't make the app work on Firefox, getting the cars returns an unauthorized response from the server.  This is just a notice, optionally take a look.